docs/user/boot_options.md

```diff
@@ -46,6 +46,15 @@ to extend.
   Setting this variable will impact all products.
   :::
 
+- `agama.finish`
+  During an unattended installation, if the installation is completed successfully then the
+  installer will reboot into the target system by default. This behavior can be modified allowing
+  to `stop`, `halt` or `poweroff` the machine at the end of the installation.
```

Comment:
NP: we don't have something like ```agama.finish=reboot``` or ```agama.finish=default``` ? Just for cases when someone want to express default behavior explicitly in his use case.